hugo/content/docs/langium-overview.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Langium Overview"
+weight: 50
+---
+
+In this chapter you'll get a closer look at the tasks that Langium aims to simplify when creating a custom domain specific language.
+As a fully featured language engineering framework, Langium supports you from parsing a string into a semantic representation of your model up until your interpreter or code generator.
```

Comment:
Some thoughts: `Semantic representation` might look weird for outstanding developers. There is also the term DDD domain-driven design (I was thinking of because of the DSLs mentioned earlier). Maybe `domain representation` suits better or even `domain model`?

P.S. for personally the domain model is a semantic model with validation done at creation time, whereas a semantic model also contains errors and validation comes after its construction. Enlight me if you have a better understanding.

Reply:
I like `domain model` here, thanks 👍